### PR TITLE
Bono/saved updates b

### PR DESCRIPTION
**Pull request by bonomite**
- capacitor libs updated
- centralized addToFavorites and dynamicNavigation
- Followed shows tab (show with 3 episodes after it)
- updated dynamicList for followed shows episodes
-

### CODE REVIEW

File: package.json
Comment (skacula) [inline package.json:1]: Was hoping these updates would be in its own commit to weed out issues.

File: components/saved/FetchRecentEpisodes.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    for (let i = 0; i < resData.length; i++) {
+      resData[i].cmsSource = cmsSources.PUBLISHER
+      resData[i] = normalizeArticlePage(humps.camelizeKeys(resData[i]))
+    }
+    //console.log(resData)
+
+    fetchedEpisodes.value = resData
+  } catch (e) {
+    console.error("getEpisodes error = ", e)
+  }
```
Comment (skacula) [inline components/saved/FetchRecentEpisodes.vue]: This call should be in BFF as this will cause some cors issues.

**Review by skacula — COMMENTED** [review]
I've made some comments on where I see some issues. The biggest one will be that fetch to publisher as that should be in the bff.

File: package.json
Comment (bonomite) [reply package.json:1]: Ah ... I tested locally and all seemed ok... They were all minor updates.

File: components/saved/FetchRecentEpisodes.vue (hunk 1 above)
Comment (bonomite) [reply components/saved/FetchRecentEpisodes.vue]: I wanted to do it in the BFF, but the slug is acquired from a slotProp on the front end... I can review what I'm doing with you and see if we can utilize the BFF.

**Review by kimlarocca — APPROVED** [review]
